frontend/e2e/$lang+/_public/apply/adult-child-flow/adult-category.spec.ts

```diff
@@ -0,0 +1,165 @@
+      await page.getByRole('group', { name: 'Are all the children you are applying for under 18?' }).getByRole('radio', { name: 'Yes' }).check();
+
+      await page.getByRole('button', { name: 'Continue' }).click();
+    });
+
+    await test.step('Should navigate to DTC page', async () => {
+      await applyAdultChildPage.isLoaded('disability-tax-credit');
+
+      await page.getByRole('radio', { name: 'No' }).check();
+      await page.getByRole('button', { name: 'Continue' }).click();
+    });
+
+    await test.step('Should navigate to apply children page', async () => {
+      await applyAdultChildPage.isLoaded('apply-children');
+      await page.getByRole('button', { name: 'Proceed to apply for your child(ren)' }).click();
+    });
+
+    await test.step('Should navigate to children application', async () => {
+      await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/child\/children/);
+      await expect(page.getByRole('heading', { level: 1, name: "Child(ren)'s application" })).toBeVisible();
+    });
+  });
+
+  test('Should navigate to adult flow if child is not eligible but applicant wish to apply for themself', async ({ page }) => {
+    const applyAdultChildPage = new PlaywrightApplyAdultChildPage(page);
+
+    await test.step('Should navigate to tax filing page', async () => {
+      await applyAdultChildPage.isLoaded('tax-filing');
+
+      await page.getByRole('radio', { name: 'Yes', exact: true }).check();
+      await page.getByRole('button', { name: 'Continue' }).click();
+    });
+
+    await test.step('Should navigate to date of birth page', async () => {
+      await applyAdultChildPage.isLoaded('date-of-birth');
+
+      const { year, month, day } = calculateDOB(35);
+
+      await page.getByRole('combobox', { name: 'Month' }).selectOption(month);
+      await page.getByRole('textbox', { name: 'Day (DD)' }).fill(day);
+      await page.getByRole('textbox', { name: 'Year (YYYY)' }).fill(year);
+      await page.getByRole('group', { name: 'Are all the children you are applying for under 18?' }).getByRole('radio', { name: 'No' }).check();
+
+      await page.getByRole('button', { name: 'Continue' }).click();
+    });
+
+    await test.step('Should navigate to DTC page', async () => {
+      await applyAdultChildPage.isLoaded('disability-tax-credit');
+
+      await page.getByRole('radio', { name: 'Yes' }).check();
+      await page.getByRole('button', { name: 'Continue' }).click();
+    });
+
+    await test.step('Should navigate to apply self page', async () => {
+      await applyAdultChildPage.isLoaded('apply-yourself');
+      await page.getByRole('button', { name: 'Proceed to apply for yourself' }).click();
+    });
+
+    await test.step('Should navigate to adult flow application', async () => {
+      await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/adult\/applicant-information/);
+      await expect(page.getByRole('heading', { level: 1, name: 'Applicant information' })).toBeVisible();
```

Comment:
Can we reference the `PlaywrightApplyAdultPage` page object model and call `isLoaded('applicant-information')`  here instead?